tests/01_air_get.R

```diff
@@ -0,0 +1,31 @@
+# Testing air_get() for exposing offset
+# Check returns variable for correct content
+
+source('tests/00_setup.R')
+source('R/airtabler.R')
+
+offset = NULL
+i = 0
+returns = list()
+repeat {
+    i = i + 1
+    
+    x = air_get(base = TEST_BASE, table_name = TEST_TABLE, offset = offset)
+
+    returns[[i]] = x
+    offset = get_offset(x)    
+
+    if ( i == 1 ) { 
+        if ( dim(x)[1] < 100 ) {
+            stop('Test air_get(): This test requires a table with at least 100 records, check .Renviron.')
```

Comment:
I wonder if it would make sense to add a test to write a large table to the test base?
For future improvement, not necessarily now.

Reply:
I think the tests should be expanded significantly, and this would take `airtabler` to the next level. It should be done in a real testing fashion: upload a local table (possibly random content with multiple types), then download it in different ways and compare them.

It would also require testing of error handling, not sure if `airtabler` has any.

I need to learn how to do proper unit tests in `R`, though...